service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -701,13 +702,19 @@ public Map<String, Object> removeCorrespondenceDocument(CaseData caseData,
                 .filter(el -> !expectedDocumentId.equals(el.getId()))
                 .collect(toList());
 
+        List<Element<SupportingEvidenceBundle>> newCorrespondenceDocumentsNC =
+            newCorrespondenceDocuments.stream()
+                .filter(el -> isNull(el.getValue().isConfidentialDocument()) || !el.getValue().isConfidentialDocument())
```

Comment:
`isNull(el.getValue().isConfidentialDocument()` will always return false because it will either return true or false and will never be null